### PR TITLE
view added food items during session

### PR DESCRIPTION
**Pull request by ZD1wang**
# Description
* created get food items during session api
* created test for get food items during session

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x]  test created in test_main.py file
- [x]  manual api testing through postman

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I will rebase before merging

### CODE REVIEW

File: backend/app/main/main.py [hunk 1]
```diff
@@ -272,3 +272,19 @@ def create_user(user: schemas.UserCreate, session: Session = Depends(get_db)):
...
+    "/get_session_food_items/{food_collect_id}", response_model=List[schemas.Food]
+)
+def get_session_food_items(food_collect_id, session: Session = Depends(get_db)):
+    """
+    Get session food items from the database - MealCare
+
+    :return: JSON response with get all entries
+    """
+
+    db_session_food_items = crud.get_session_food_items(session, food_collect_id)
```
Comment (Ty-Won) [inline backend/app/main/main.py:287]: nice job, maybe just change the crud method call to match the rest of the endpoints i.e)crud.get_session_food_items(session: Sessio, food_collect_id=food_collect_id)

**Comment by Ty-Won** [comment]
I notice the way you wrote your test is like the rest of the tests which is good.  One thing to note is that every test i notice is creating a vendor. In the future it might be good to rework this to include a single vendor at the top that applies for all and modify that specific vendor in place. This is just a nitpick though

**Comment by ZD1wang** [comment]
> I notice the way you wrote your test is like the rest of the tests which is good. One thing to note is that every test i notice is creating a vendor. In the future it might be good to rework this to include a single vendor at the top that applies for all and modify that specific vendor in place. This is just a nitpick though

Definitely, I actually brought this up before. We should create a ticket for this.

**Comment by Ty-Won** [comment]
@ZD1wang  definitely agree with you

**Pull request merged**